nano/core_test/node.cpp

```diff
@@ -1682,25 +1682,43 @@ TEST (node, bootstrap_confirm_frontiers)
 }
 
 // Test that if we create a block that isn't confirmed, we sync.
-TEST (node, DISABLED_unconfirmed_send)
+TEST (node, unconfirmed_send)
```

Comment:
Could you write an overview paragraph here explaining what the test does overall?

Reply:
On seconds thoughts,  it is fine, the inline comments are good enough to understand what is happening.